README.md

```diff
@@ -42,6 +42,45 @@ Explore our [collaborative examples](https://github.com/drifting-in-space/y-swee
 
 All examples are open source and live in this repository, within [/examples](https://github.com/drifting-in-space/y-sweet/tree/main/examples).
 
+## Usage
+
+Check the [vanilla js example](/examples/vanilla/) for more details.
+
+### On the server
+``` js
+import { createDoc } from '@y-sweet/sdk'
+// Get the client token from the y-sweet server. This is basically the user's "password" to edit the "myDoc123" doc.
+app.get('/client-token', async (req, res) => {
+  const clientToken = await createDoc('myDoc123')
```

Comment:
This should include a `CONNECTION_STRING` I think?

Reply:
i dont think createDoc requres a connection string, so was thinking it would be simpler to drop that entirely and rely on other tutorials in the docs to cover the concept of the connection string (just cause explaining it here would mean a big chunk of text

Reply:
I'm realizing now that `createDoc('id123', CONNECTION_STRING)` only returns a `DocCreationResult`. So we'll need to call `getClientToken(id, CONNECTION_STRING)` right after to get the client token. And because we'd have to pass both a `CONNECTION_STRING`, maybe using the `DocumentManager` API is the clearest / simplest?

e.g.:

```ts
const manager = new DocumentManager(CONNECTION_STRING)
const { doc } = await manager .createDoc('myDoc123')
const clientToken = await manager.getClientToken(doc, {})
```

Reply:
just added this lmk what you think!